CMakeLists.txt

```diff
@@ -0,0 +1,89 @@
+cmake_minimum_required(VERSION 3.14)
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake")
+include(prelude)
+
+project(
+  Qt-Frameless-Window-DarkStyle
+  VERSION 0.1.0
+  DESCRIPTION "simple MainWindow class implementation with frameless window and custom dark style. "
+  HOMEPAGE_URL "https://github.com/Jorgen-VikingGod/Qt-Frameless-Window-DarkStyle"
+  LANGUAGES CXX
+)
+
+include(project-is-top-level)
+include(variables)
+include(link_qt)
+
+qt5_add_resources(
```

Comment:
What about qt6?

Reply:
Got it

Reply:
Can you believe even now in 2022 our products have been targeting on Windows XP o(╥﹏╥)o